Building according to README throws Typescript error TS7026

I was attempting to create a PR to fix #133, but I can't build the project due to the following errors:

```
node_modules/vega-lite/build/src/log.d.ts(4,33): error TS7016: Could not find a declaration file for module 'vega-util'. '/Users/.../vega-tooltip/node_modules/vega-lite/node_modules/vega-util/build/vega-util.js' implicitly has an 'any' type.
node_modules/vega-lite/build/src/log.d.ts(14,33): error TS7016: Could not find a declaration file for module 'vega-util'. '/Users/.../vega-tooltip/node_modules/vega-lite/node_modules/vega-util/build/vega-util.js' implicitly has an 'any' type.
node_modules/vega-lite/build/src/util.d.ts(2,93): error TS7016: Could not find a declaration file for module 'vega-util'. '/Users/.../vega-tooltip/node_modules/vega-lite/node_modules/vega-util/build/vega-util.js' implicitly has an 'any' type.
src/supplementField.ts(161,5): error TS2322: Type '{}' is not assignable to type 'FieldDef<string>'.
  Property 'type' is missing in type '{}'.
```

I haven't used Typescript before, perhaps I'm missing something in my setup? I have run `npm i -g typescript`.

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -5,7 +5,7 @@
     "moduleResolution": "node",
     "isolatedModules": false,
     "declaration": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
     "noEmitOnError": true,
     "noImplicitThis": true,
     "noImplicitReturns": true,

